types/estree-jsx/index.d.ts

```diff
@@ -29,10 +29,14 @@ export interface JSXIdentifier extends BaseNode {
 
 export interface JSXMemberExpression extends BaseExpression {
     type: 'JSXMemberExpression';
+    object: JSXMemberExpression | JSXIdentifier;
```

Comment:
For testing these additions, you should be able to add something similar to the tests for `JSXAttribute`:
```ts
// JSXMemberExpression
const expressionObject: JSXMemberExpression | JSXIdentifier = memberExpression.object;
...
```